### PR TITLE
h2object brew install request

### PR DESCRIPTION
**Pull request by liujianping**
first version 0.0.1.beta commit

### CODE REVIEW

File: Library/Formula/h2object.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require 'formula'
```
Comment (bfontaine) [inline Library/Formula/h2object.rb]: Please remove this line

File: Library/Formula/h2object.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+class H2object < Formula
+  homepage 'http://h2object.io'
```
Comment (bfontaine) [inline Library/Formula/h2object.rb]: Use double quotes everywhere.

File: Library/Formula/h2object.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  url 'http://dl.h2object.io/h2object/macosx/h2object-0.0.1-beta.tar.gz'
```
Comment (bfontaine) [inline Library/Formula/h2object.rb]: Is there a stable release we can use? It’ll need to go to [Homebrew/homebrew-devel-only](https://github.com/homebrew/homebrew-devel-only) if there is only a beta release.

File: Library/Formula/h2object.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  sha1 '588bab3d9372b433d15616a8b8bbe6d2720f27d6'
```
Comment (bfontaine) [inline Library/Formula/h2object.rb]: `sha1`s are deprecated, please use `sha256`.

```
$ curl -sL http://dl.h2object.io/h2object/macosx/h2object-0.0.1-beta.tar.gz | shasum -a 256
262f2f491baca8c06f600f2c94506592a08cbfc6fede51e53586b58d3e8c0fe5  -
```

File: Library/Formula/h2object.rb [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+  def install
+    bin.install 'h2object'
+  end
```
Comment (bfontaine) [inline Library/Formula/h2object.rb]: Could you add a test? We need something to ensure it works, so it should be more substantial than e.g. `--help` or `--version`. Thanks!

**Comment by tdsmith** [comment]
This may not meet our notability requirements; this seems like a very young project. Is the source code available? Software in Homebrew/homebrew must be built from source. How is it licensed?

**Comment by liujianping** [comment]
source code now is not open temporarily for more readable reasons, will use GPL license recently.

**Comment by DomT4** [comment]
We'll need the source code available before considering it, in all likelihood. Given the `beta` tag it may be a better fit for the `devel-only` tap temporarily.

**Comment by mikemcquaid** [comment]
Passing on this until source code is available and it's more clear that this is a popular, established project.

**Pull request closed without merging**